docker-compose.yml

```diff
@@ -2,13 +2,15 @@ version: "3.0"
 services:
 
   database:
-    image: postgres:10
+    image: amsterdam/postgres
     ports:
```

Comment:
De postgres versie MOET 10 of hoger zijn, anders werkt de full-text search niet. Als de Amsterdam postgres docker nu versie 10 heeft vind ik dit goed. Heb je dat gecontroleerd?